orangecontrib/prototypes/widgets/owunique.py

```diff
@@ -61,30 +60,21 @@ def __init__(self):
     def set_data(self, data):
         self.closeContext()
         self.data = data
+        self.selected_vars = []
```

Comment:
This line is the actual fix. Everything else is about removing summaries. I accidentaly squashed the two commits and I'm too lazy to separate them (because removed lines 71 and 72 belong to different commits :).